test/manual/socks5.test.ts

```diff
@@ -302,7 +302,7 @@ describe('Socks5 Connectivity', function () {
 async function testConnection(connectionString, clientOptions) {
   const client = new MongoClient(connectionString, clientOptions);
   let topologyType;
-  client.on('topologyDescriptionChanged', ev => (topologyType = ev.newDescription.type));
+  client.on('topologyDescriptionChanged', ev => (topologyType = ev.previousDescription.type));
```

Comment:
This function was previously relying on the fact that the last `TopologyDescroptionChangedEvent` would be the one transitioning the topology to the fully known, desired operating state, with the description being in the `newDescription` field.

Now that we emit a final instance of that event transitioning to the `UNKNOWN` state, this final event would have the desired topology description in the `previousDescription` field, rather than the `newDescription` field.